packages/card/Readme.md

```diff
@@ -11,7 +11,7 @@ const CardActions = require('./src').CardActions;
     Here come actions
   </CardActions>
   <CardBody>
-    <CardIcon icon="fa-bell" ariaLabel="This is a bell icon" />
+    <CardIcon icon="ai-alarm-bell" ariaLabel="This is a bell icon" />
```

Comment:
Want hier staat er wel ai- voor de alarm-bell

Reply:
Yes, ik heb het voor de consistentie in de code overal zelf geprefixed, maar in de testen wou ik kijken of het automatisch gebeurde indien het prefix werd weggelaten.